src/asm/x86/transform/inverse.rs

```diff
@@ -247,3 +247,125 @@ decl_tx_fns! {
   rav1e_inv_txfm_add_identity_flipadst_4x4_ssse3,
   rav1e_inv_txfm_add_flipadst_identity_4x4_ssse3
 }
+
+#[cfg(test)]
+#[rustfmt::skip]
+mod test {
+  use super::*;
+  use crate::frame::{AsRegion, Plane};
+  use crate::transform::TxSize::*;
+  use rand::random;
+
+  macro_rules! decl_tx_fn_test {
+    ($ENUM:pat, $TYPE1:ident, $TYPE2:ident, $W:expr, $H:expr, $OPT:ident, $OPTLIT:literal) => {
```

Comment:
The list of test cases would be a little nicer if this macro accepted a list. Then we might call it `test_itx_fns`.